src/components/chat/DyadWebCrawl.tsx

```diff
@@ -0,0 +1,27 @@
+import type React from "react";
+import type { ReactNode } from "react";
+import { Rabbit } from "lucide-react";
+
+interface DyadWebCrawlProps {
+  children?: ReactNode;
+  node?: any;
+}
+
+export const DyadWebCrawl: React.FC<DyadWebCrawlProps> = ({
+  children,
+  node: _node,
+}) => {
+  return (
+    <div className="bg-(--background-lightest) rounded-lg px-4 py-2 border my-2">
+      <div className="flex items-center justify-between">
+        <div className="flex items-center gap-2">
+          <Rabbit size={16} className="text-blue-600" />
```

Comment:
can we use this icon instead? https://lucide.dev/icons/scan-qr-code something a little more relevant than a rabbit.